extensions/wikia/SpecialPageViews/modules/SpecialPageViews.js

```diff
@@ -0,0 +1,478 @@
+var SponsorshipDashboard = function(){
```

Comment:
This file require some linting work but I'm not sure is it worth, because of incoming new chart library investigation.